docs/running_on_staging.md

```diff
@@ -1,18 +1,28 @@
-## Running Your Own Subtensor Chain Locally
+# Running Subnet Locally
 
-This tutorial will guide you through setting up a local subtensor chain, creating a subnetwork, and connecting your mechanism to it.
+This tutorial will guide you through setting up a local blockchain (subtensor), creating a subnet, and run your incentive mechanism on the subnet.
 
-### 1. Install substrate dependencies
+## Prerequisites
+
+Before proceeding further, make sure that you have installed Bittensor and the Wallet. See the below instructions:
+
+- [Install `bittensor`](https://github.com/opentensor/bittensor#install).
+- [Install `wallet`](https://github.com/opentensor/bittensor#wallets).

```

Comment:
Install -> Create wallet.

We used to have the wallet separated out into its own package, so this language could cause some confusion.